source/computer/wsl-setup.rst

```diff
@@ -1,18 +1,212 @@
 WSL 配置指南
 ============
 
+:本节贡献者: |赵志远|\（作者）、
+             |姚家园|\（审稿）、
+             |田冬冬|\（审稿）
+:最近更新日期: 2021-01-18
+:预计花费时间: 120 分钟
+
+.. warning::
+
+   本文尚未经过除作者外的其他 Windows 用户测试，欢迎阅读并给与反馈。
+
+简介
+----
+
+WSL 有 WSL1 和 WSL2 两个发行版本，二者底层原理不同。大多数情况下，建议使用 WSL2，
+因为它提供更快的性能和 100% 的系统调用兼容性。涉及到跨系统的文件互访时
+（Linux 访问 Windows 里的文件，或 Windows 访问 Linux 里的文件），使用 WSL1 具有
+更快的性能。
+
+官方目前没有弃用 WSL1 的计划，并且支持将任何一个已经安装的 Linux 发行版转换为 WSL1 或者 WSL2。
+
+安装 WSL
+--------
+
+WSL 的安装方式可以参考官方文档：
+
+- 中文指南：https://docs.microsoft.com/zh-cn/windows/wsl/install-win10
+- 英文指南：https://docs.microsoft.com/en-us/windows/wsl/install-win10
+
+官方文档对 WSL 安装配置中可能遇到的常见问题都给出了详细的解决方案，请认真阅读。
+
+WSL 可以安装不同的 Linux 发行版，但目前官方并未提供 Fedora 发行版的安装。
+推荐使用 WSL 安装 Ubuntu 20.04 LTS。
+
 .. note::
 
-   尚未完成。
+   想在 WSL 上安装 Fedora 的读者可以参考以下安装指南：
+
+   - 英文指南：https://fedoramagazine.org/wsl-fedora-33/
+   - 中文指南：https://suiahae.me/Using-Fedora-33-on-Windows-10-WSL2/
+   
+   指南中的 Fedora rootfs 下载地址可能已失效。请访问 
+   https://github.com/fedora-cloud/docker-brew-fedora/tree/33/x86_64
+   下载最新版本。
+      
```

Comment:
```suggestion
.. warning::

   若启用 WSL，则其他虚拟机软件（如 VMware 和 VirtualBox）可能会无法使用。
```

@zhaozhiyuan1989 Is this what you meant?

Reply:
Only WSL2.

Reply:
Updated